docs/crossgen-scenarios.md

```diff
@@ -0,0 +1,50 @@
+
+# Crossgen Scenarios
+An introduction of how to run scenario tests can be found in [Scenarios Tests Guide](./scenarios-workflow.md).  The current document has specific instruction to run:
+
+- [Crossgen Throughput](#crossgen-throughput)
+- [Crossgen2 Throughput](#crossgen2-throughput)
+
+
+## Crossgen Throughput
+
+### Step 0 Generate Core Root
+Build CoreCLR in dotnet/runtime repo, which creates Core Root. Instruction can be found [here](https://github.com/dotnet/runtime/blob/master/docs/workflow/building/coreclr/README.md). A detailed introduction of Core Root can be found [here](https://github.com/dotnet/runtime/blob/master/docs/workflow/testing/using-corerun.md).
+### Step 1 Initialize Environment
+Same instruction of [Step 1 in Scenario Tests Guide](./scenarios-workflow#step-1-initialize-environment).
+### Step 2 Run Precommand
+```
+cd crossgen
+# No precommand needed for this scenario
+```
+### Step 3 Run Test
+```
+py -3 test.py crossgen --core-root <path to Core Root directory> --test-name <assembly to compile>
+```
+The above command runs the test. Note `--test-name <assembly to compile>` option refers to the relative path of an assembly that's under Core Root directory. For example, the option can be `--test-name System.Private.Xml.dll` so the test measures the throughput of crossgen compiling `System.Private.Xml.dll`.
+### Step 4 Run Postcommand
+Same instruction of [Step 4 of Scenario Tests Guide](./scenarios-workflow#step-4-run-postcommand).
+
+## Crossgen2 Throughput
+Refer to [Crossgen Throughput](crossgen-throughput) for Step 0,1 and 4
+### Step 2 Run Precommand
+```
+cd crossgen2
+# No precommand needed for this scenario
+```
+### Step 3 Run Test
+For scenario which compiles a **single assembly**:
```

Comment:
It would be helpful to add a link to public doc for explanation of "single assembly", like https://docs.microsoft.com/en-us/dotnet/core/deploying/single-file? or add it to the relavant links section.

Reply:
It means a single `.dll` file name under `Core_Root`. Example added to the help message.